src/lipalightninglib.udl

```diff
@@ -278,6 +281,13 @@ dictionary InvoiceDetails {
     timestamp expiry_timestamp; // The moment an invoice expires (UTC)
 };
 
+// Indicates the max fee that can potentially be paid
+[Enum]
+interface MaxFeeStrategy {
```

Comment:
We can try to find a better word, but my goal was to make this enum represent the "lower-level" strategy applied for a payment of a specific size. Not the overall max fee strategy. I wouldn't call this `...Fee` because we don't necessarily want to represent a max fee. We want to represent the logic used to get that max fee.